redwood-lazylayout-view/src/main/kotlin/app/cash/redwood/lazylayout/view/ViewLazyList.kt

```diff
@@ -286,11 +274,12 @@ internal open class ViewLazyList(context: Context) : LazyList<View> {
         }
         is ViewHolder.Item -> {
           val index = position - items.itemsBefore
-          val view = items.widgets[index].value
+          val widget = items.widgets[index]
           holder.container.removeAllViews()
-          (view.parent as? FrameLayout)?.removeAllViews()
-          view.layoutParams = layoutParams
-          holder.container.addView(view)
+          (widget.value.parent as? FrameLayout)?.removeAllViews()
+          widget.value.layoutParams = layoutParams
+          holder.container.addView(widget.value)
+          widget.value.applyModifier(widget.modifier)
```

Comment:
This needs to be re-applied if the modifier changes. I tried listening to changes in `Items#onModifierUpdated`, but it was never called on change.

Reply:
Seems like a bug! FYI @JakeWharton